src/style.css

```diff
@@ -1 +1,44 @@
-/* Styles go here */
+* {
+  margin: 0;
+  padding: 0;
+}
```

Comment:
use `body` tag here instead of `*`, as this has problem for certain elements to inherit from this
for certain properties like `box-sizing` you should use `*` but not here